app/main.py

```diff
@@ -1,8 +1,59 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class, clean_mark, brand):
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center,
+            clean_power,
+            average_rating,
+            count_of_ratings
+    ):
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car):
+
+        # return round((car.comfort_class * (self.clean_power - car.clean_mark)
+        #               * self.average_rating
+        #               / self.distance_from_city_center), 1
+        #              )
+        cc = car.comfort_class
+        cp = self.clean_power
+        cm = car.clean_mark
+        ar = self.average_rating
+        dis = self.distance_from_city_center
```

Comment:
it's a bad code. You can add W504 flake8 rule to ignored ones or create temporal variables with good names with intermediate calculations